recipes/salabim/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "salabim" %}
+{% set version = "24.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/salabim-{{ version }}.tar.gz
+  sha256: f76a91496eebcf8c02ca70fc9380a7bffc652156589c82d1726c4f11ad45e19a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+  run_constrained:
+    - greenlet
+    - numpy
+    - opencv
+    - pandas
+    - pillow
+    # salabim docs say this stricth pin is needed, but worried that this will
+    # be problematic in environment solves
+    - pyglet  # ==1.5.27
+    - pyopengl
+    - pysimplegui
+    - python-dateutil
+    - scipy
+    # no feedstock, but may be added in future
+    # - pywavefront
```

Comment:
Why does pip check work if this isn't available? Is it an optional dependency?

Reply:
They all optional dependencies that are not specified programmatically upstream. They are only listed in the documentation.